packages/react-internal/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -34,7 +34,9 @@ export enum KeyboardSpinDirection {
 /**
  * {@docCategory SpinButton}
  */
-export interface ISpinButtonProps extends React.HTMLAttributes<HTMLDivElement>, React.RefAttributes<HTMLDivElement> {
+export interface ISpinButtonProps
+  extends React.HTMLAttributes<HTMLInputElement | HTMLTextAreaElement>,
```

Comment:
HTMLTextAreaElement shouldn't be included in the props here. If this was copied from TextField, it's only included there because TextField actually can be an `input` or a `textarea` internally (but SpinButton can only be an `input`).